shotgun_api3/shotgun.py

```diff
@@ -278,6 +283,7 @@ def __init__(self,
                  ca_certs=None,
                  login=None,
                  password=None,
+                 auth_token=None,
```

Comment:
New optional parameters should be added at the end in case anybody is using the constructor passing in arguments by position.  Otherwise it is a non-backward compatible change.